elasticsearch/includes/classes/class-elasticsearch.php

```diff
@@ -54,6 +54,9 @@ protected function setup_hooks() {
 		add_filter( 'ep_intercept_remote_request', '__return_true', 9999 );
 		add_filter( 'ep_do_intercept_request', [ $this, 'filter__ep_do_intercept_request' ], 9999, 4 );
 		add_filter( 'jetpack_active_modules', [ $this, 'filter__jetpack_active_modules' ], 9999 );
+
+		//Filter jetpack widgets
+		add_filter( 'jetpack_widgets_to_include', [ $this, 'filter__jetpack_widgets_to_include' ], 10, 1 );
```

Comment:
```suggestion
		add_filter( 'jetpack_widgets_to_include', [ $this, 'filter__jetpack_widgets_to_include' ], 10 );
```

Can leave off the args count param if there is just 1.